maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -437,16 +437,7 @@ class Maestro(private val driver: Driver) : AutoCloseable {
         private const val SCREENSHOT_DIFF_THRESHOLD = 0.005 // 0.5%
         private const val ANIMATION_TIMEOUT_MS: Long = 15000
 
-        fun ios(host: String, port: Int): Maestro {
-            val channel = ManagedChannelBuilder.forAddress(host, port)
-                .usePlaintext()
-                .build()
-
-            return ios(channel)
-        }
-
-        fun ios(channel: ManagedChannel): Maestro {
-            val driver = IOSDriver(IdbIOSDevice(channel))
+        fun ios(driver: Driver): Maestro {
```

Comment:
Maestro already have a constructor that takes `Driver` as a parameter. This is not much different except that it calls `driver.open()`

Why not have `ios()` method actually create a driver internally?

Reply:
`Maestro` already has a constructor that takes `Driver` as a parameter, we can just use that constructor for Cloud